### PR TITLE
Enhance branching

### PR DESCRIPTION
**Pull request by jkonecny12**
Add specific branching configuration file.
Enhance branching guide.

It would be great to have feedback on the branching guide. I was struggling how to adapt current guide to make the changes.

This guide should be validated during the next branching.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -204,6 +205,48 @@ ci:
...
+
+	@echo "===================================="
+	@echo "Testing localization for $(L10N_DIR)"
+	@echo "===================================="
+	@curl --fail --silent --show-error -L https://raw.githubusercontent.com/rhinstaller/anaconda-l10n/$(GIT_L10N_BRANCH)/$(L10N_DIR)/anaconda.pot >/dev/null ||  \
+		(echo "*** Can't download translations from a localization repository"; \
+		 echo "repository: $(L10N_REPOSITORY)"; \
+		 echo "branch:     $(GIT_L10N_BRANCH)"; \
+		 echo "directory:  $(L10N_DIR)"; \
+	   exit 2)
```
Comment (VladimirSlavik) [inline Makefile.am]: Indentation... ;-)

Comment (jkonecny12) [reply Makefile.am]: Broken because I'm messing with tabs and spaces... Hmm I should be consistent.

Comment (VladimirSlavik) [reply Makefile.am]: If that's the only problem with the code, you're doing something right!

Comment (jkonecny12) [reply Makefile.am]: You know, it could be because people saw that bashisms everywhere and rather avoid looking on the code :D

Comment (jkonecny12) [reply Makefile.am]: Fixed.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Fix indentation

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Pull request merged**